sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorConverter.cs

```diff
@@ -36,23 +36,28 @@ internal static class AzureMonitorConverter
             [TelemetryType.Event] = "Event",
         };
 
-        public static List<TelemetryItem> Convert(Batch<Activity> batchActivity, string instrumentationKey)
+        internal static List<TelemetryItem> Convert(Batch<Activity> batchActivity, string instrumentationKey)
         {
             List<TelemetryItem> telemetryItems = new List<TelemetryItem>();
             TelemetryItem telemetryItem;
 
-            foreach (var activity in batchActivity)
+            // TODO: Batch<Activity> batchActivity = new Batch<Activity>() creates a null entry with no activity.
```

Comment:
Is `new Batch<Activity>()` allowed? It might get fixed in the latest version and upcoming 0.7.0-beta.
The correct way is to use `new Batch<Activity>(activity)`.

Reply:
Just realized after reviewing code in OTel repo. Will get the condition removed.

Reply:
If `new Batch<Activity>()`, it will be a bug for OpenTelemetry .NET and please file an issue.